### PR TITLE
FAQ: Give credit to p0nce for Kakoune logo

### PR DESCRIPTION
**Pull request by alexherbo2**

### CODE REVIEW

File: doc/pages/faq.asciidoc [hunk 1]
```diff
@@ -6,6 +6,12 @@ The name of the project is pronounced "Kak-oon", and is a word taken from a
 New Caledonian dialect based on french. It means a hard blow, usually a punch,
 but generally refers to a blow into which all of one's strength went.
 
+== Who wrote the Kakoune logo?
```
Comment (maximbaz) [inline doc/pages/faq.asciidoc]: "made" instead of "wrote"

**Comment by lenormf** [comment]
As we discussed on IRC, I thought the credits could be more visible so I made a PR that adds a "Credits" section to the README, and names @p0nce in the logo's `title` attribute.

**Comment by p0nce** [comment]
Thanks.
It's kinda my most successful logo http://www.gamesfrommars.fr/hand-made-svg-logos/

**Comment by alexherbo2** [comment]
@lenormf You should add yourself for the site.  Although you told me on IRC you don’t mind about the credits for yourself, I think it’s a valuable information to know how things got crafted.

**Comment by alexherbo2** [comment]
Closing for #2478.

**Comment by lenormf** [comment]
I actually have a `meta` tag set to my name (attribute: `author`) on the site, that's as visible as I'd like the name to be :)

**Pull request closed without merging**